coalib/__init__.py

```diff
@@ -1,3 +1,10 @@
+"""
```

Comment:
you are wrapping the commit body weirdly in a way thats unreadable. please wrap at most at 80 chars, but not waaay less than 80 as you did. also end with period after "documentation"